apps/site/assets/ts/models/alert.ts

```diff
@@ -159,24 +159,33 @@ const isCurrentLifecycle = ({ lifecycle }: Alert): boolean =>
   lifecycle === "ongoing" ||
   lifecycle === "ongoing_upcoming";
 
-export const isCurrentAlert = (
+export const isBannerAlert = ({ effect }: Alert): boolean =>
+  ["suspension", "stop_closure", "station_closure", "shuttle"].includes(effect);
```

Comment:
suggestion(nit-pick): This is for testing if its a banner alert for the stop page. I'd move this logic to the stop page as that is where this code makes sense.  Other pages may have different alerts for the banners.  Also we have a website wide banner alert this may be confused with for testing.